_data/drawings.yaml

```diff
@@ -1,3 +1,42 @@
-- title: Number ONE
-  description: Numero uno, little Juan
-  img_file: num1.png
+- title: Dad (2021)
+  description: 29.7 x 42cm, Pencil On Paper
+  img_file: Dad 2021.jpeg
```

Comment:
I just checked and I'm afraid none of these images work - you've used '.jpeg' when they're all '.jpg' hah